t/test.t

```diff
@@ -75,7 +75,67 @@ my $critic = Perl::Critic->new(
 
     my @violations = $critic->critique( \$str );
 
-    is( scalar @violations, 0 );
+    is( scalar @violations, 0, "No bug - with or and parenthesis and whitespace");
+}
+
+# The bug with two-arg open
+{
+    my $str = q[
+        open my $fh, "<$file"
+            || die "Can't open '$file': $!";
+    ];
+
+    my @violations = $critic->critique( \$str );
+
+    is( scalar @violations, 1, "Two-arg open bug");
+}
+
+# No bug - with parenthesis
+{
+    my $str = q[
+        open(my $fh, "<$file")
+            || die "Can't open '$file': $!";
+    ];
+
+    my @violations = $critic->critique( \$str );
+
+    is( scalar @violations, 0, "No bug - with parenthesis");
+}
+
+# No bug - with or
+{
+    my $str = q[
+        open my $fh, "<$file"
+            or die "Can't open '$file': $!";
+    ];
+
+    my @violations = $critic->critique( \$str );
+
+    is( scalar @violations, 0, "No bug - with or");
+}
+
+# No bug - with or and parenthesis
+{
+    my $str = q[
+        open(my $fh, "<$file")
+            or die "Can't open '$file': $!";
+    ];
+
+    my @violations = $critic->critique( \$str );
+
+    is( scalar @violations, 0, "No bug - with or and parenthesis");
+}
+
+# No bug - with or and parenthesis and whitespace
+{
+    my $str = q[
+        open ( my $fh, "<$file" )
+            || die "Can't open '$file': $!";
+    ];
+
+    my @violations = $critic->critique( \$str );
+
+    is( scalar @violations, 0, "No bug - with or and parenthesis and whitespace");
```

Comment:
Please use single-quotes for non-interpolated strings as described in [ProhibitInterpolationOfLiterals](https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitInterpolationOfLiterals)

```suggestion
    is( scalar @violations, 0, 'No bug - with or and parenthesis and whitespace' );
```